### PR TITLE
Overhauling tank drone

### PR DESCRIPTION
**Pull request by Regularitee**
As per proposal in https://github.com/CleverRaven/Cataclysm-DDA/issues/25321, tripling armor and removing 120mm cannon and retaining the 40mm long-range/anti-armor role, at least until such a time other vehicle-grade weapons can be mainlined from the tank mod.

### CODE REVIEW

File: data/json/monsters.json [hunk 1]
```diff
@@ -2405,8 +2405,8 @@
   {
     "id": "mon_tankbot",
     "type": "MONSTER",
-    "name": "tank drone",
-    "description": "The Northrup Emancipator is the first and only automated tank ever produced, made shortly after the split-up of Northrup Grumman.  Clad in depleted uranium plating, and armed with advanced munitions and a 120-mm gun, it is capable of delivering extraordinary firepower.",
+    "name": "Beagle Mini-Tank UFV",
+    "description": "The Northrup Beagle is a tankette-sized urban warfare UGV. Sporting an anti-tank missile launcher, 40mm grenade launcher, and numerous anti-infantry weapons, it's designed for high-risk urban fighting.",
```
Comment (kevingranade) [inline data/json/monsters.json]: Needs to go smaller, it's still not tankette-sized as long as it's single tile.

Comment (Reclusive-reptile) [reply data/json/monsters.json]: UFV is an unmanned flying vehicle.

File: data/json/monsters.json [hunk 2]
```diff
@@ -2405,8 +2405,8 @@
   {
     "id": "mon_tankbot",
     "type": "MONSTER",
-    "name": "tank drone",
-    "description": "The Northrup Emancipator is the first and only automated tank ever produced, made shortly after the split-up of Northrup Grumman.  Clad in depleted uranium plating, and armed with advanced munitions and a 120-mm gun, it is capable of delivering extraordinary firepower.",
+    "name": "Beagle Mini-Tank UGV",
+    "description": "The Northrup Beagle is a tankette-sized urban warfare UGV. Sporting an anti-tank missile launcher, 40mm grenade launcher, and numerous anti-infantry weapons, it's designed for high-risk urban fighting.",
```
Comment (mlangsdorf) [inline data/json/monsters.json]: Replace the end of the first sentence in the description with " warfare Urban Ground Vehicle (UGV). " There should also be two spaces after the '.' according to CDDA's manual of style.

Comment (Regularitee) [reply data/json/monsters.json]: Isn't "Unmanned Ground Vehicle" the standard expansion of [UGV](https://www.google.ca/search?q=ugv&oq=ugv&aqs=chrome..69i57j0l5.535j0j7&sourceid=chrome&ie=UTF-8)? And I'll fix the typos.

**Pull request merged**